source/MapDetailPanel.cpp

```diff
@@ -592,7 +592,7 @@ void MapDetailPanel::DrawKey()
 		};
 		for(int i = 0; i < 3; ++i)
 		{
-			RingShader::Draw(pos, OUTER, INNER, MapColor(1 - i));
+			RingShader::Draw(pos, OUTER, INNER, MapColor(i - 1));
```

Comment:
I don't think this is the correct part to edit.
This code draws the rings in the key for the "visited planets" overlay.
But, the appearance of that mode is not being changed here.
```suggestion
			RingShader::Draw(pos, OUTER, INNER, MapColor(1 - i));
```